recipes/rosdep/recipe/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "rosdep" %}
+{% set version = "0.15.2" %}
+{% set sha256 = "8cb8d2df308951bea2824597194e4b34b73a22a05b462e92a6859bc8bdf6ac90" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ros-infrastructure/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Can be this be `noarch: python`?
```suggestion
  number: 0
  noarch: python
```

---

unfortunately not, since it uses the `scripts` option in setup.py: https://github.com/ros-infrastructure/rosdep/blob/master/setup.py#L14

---

Maybe add this as a comment in future recipes